ios/FluentUI/Tooltip/TooltipPositionController.swift

```diff
@@ -84,7 +87,7 @@ class TooltipPositionController {
         return CGRect(origin: tooltipOrigin, size: tooltipSize)
     }
 
-    private lazy var tooltipOrigin: CGPoint = {
+    private var tooltipOrigin: CGPoint {
```

Comment:
why can't it be lazy?

Reply:
lazy can't be used on a computed property. we need tooltipOrigin to be recalculated each time it is called to adjust to new anchor position

Reply:
i guess that the part i am confused. either this is source of truth and computation should be done somewhere else as a function?

Reply:
can you explain more? isn't the point of a computed property for it to be computed? why should that be separated into two parts (the var and a func to compute)?

Reply:
we have 3 things tooltiprect/tooltiporigin/idealtooltiporigin. My assumption is at the end tooltipRect.origin is the source of truth for tooltipviewcontroller. now that tooltiporigin is meant to be computational i don't know why this is yet another variable? to me it should be more a function?